docs/readthedocs/source/doc/Nano/Overview/pytorch_train.md

```diff
@@ -1,102 +1,224 @@
 
-```python
-from bigdl.nano.pytorch import TorchNano
 
-class MyNano(TorchNano) :
-    def train(self, ...):
-        # copy your train loop here and make a few changes
+```eval_rst
+
+.. tabs::
+
+    .. tab:: Trainer
+
+        .. code-block:: python
+
+            from bigdl.nano.pytorch import Trainer
+
+            trainer = Trainer(max_epochs=5, precision='bf16')
+            trainer.fit(...)
+
+    .. tab:: TorchNano
+
+        .. code-block:: python
+
+            from bigdl.nano.pytorch import TorchNano
+
+            class MyNano(TorchNano) :
+                def train(self, ...):
+                    # copy your train loop here and make a few changes
+
+            MyNano(precision='bf16').train(...)
 
-MyNano().train(...)
 ```
 
-Our `TorchNano` also integrates IPEX and distributed training optimizations. For example,
+### Channels Last Memory Format
+`bigdl.nano.pytorch.Trainer` API extends PyTorch Lightning Trainer with multiple integrated optimizations. You could instantiate a BigDL-Nano Trainer with channels_last=True to use the channels last memory format, i.e. NHWC (batch size, height, width, channels), as an alternative way to store tensors in classic/contiguous NCHW order.
 
-```python
-from bigdl.nano.pytorch import TorchNano
+```eval_rst
 
-class MyNano(TorchNano):
-    def train(self, ...):
-        # define train loop
+.. tabs::
+
+    .. tab:: Trainer
 
-# enable IPEX optimizaiton
-MyNano(use_ipex=True).train(...)
+        .. code-block:: python
+
+            from bigdl.nano.pytorch import Trainer
+
+            trainer = Trainer(max_epochs=5, channels_last=True)
+            trainer.fit(...)
+
+    .. tab:: TorchNano
+
+        .. code-block:: python
+
+            from bigdl.nano.pytorch import TorchNano
+
+            class MyNano(TorchNano) :
+                def train(self, ...):
+                    # copy your train loop here and make a few changes
+
+            MyNano(channels_last=True).train(...)
 
-# enable IPEX and distributed training, using 'subprocess' backend
-MyNano(use_ipex=True, num_processes=2, distributed_backend="subprocess").train(...)
 ```
 
-### Optimized Data Pipeline
 
-Computer Vision task often needs a data processing pipeline that sometimes constitutes a non-trivial part of the whole training pipeline. Leveraging OpenCV and libjpeg-turbo, BigDL-Nano can accelerate computer vision data pipelines by providing a drop-in replacement of torch_vision's `datasets` and `transforms`.
+## Accelerate `torchvision` data processing
+
+Computer Vision task often needs a data processing pipeline that sometimes constitutes a non-trivial part of the whole training pipeline.
+
+Leveraging OpenCV and libjpeg-turbo, BigDL-Nano can accelerate computer vision data pipelines by providing a drop-in replacement of `torchvision`'s components such as `datasets` and `transforms`. Nano provides a patch API `patch_torch` to accelerate these functions.
```

Comment:
I think it might be better we expose this side effect to users in our trouble shooting? Different workaround fits different users with various target.